src/structures/ApplicationCommand.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Base = require('./Base');
+const { ApplicationCommandOptionType } = require('../util/Constants');
+const Snowflake = require('../util/Snowflake');
+
+/**
+ * Represents an application command, see {@link InteractionClient}.
+ * @extends {Base}
+ */
+class ApplicationCommand extends Base {
+  constructor(client, data, guildID) {
+    super(client);
+
+    /**
+     * The ID of the guild this command is part of, if any.
+     * @type {?Snowflake}
+     * @readonly
+     */
+    this.guildID = guildID || null;
```

Comment:
Reviving the suggestion that was done here before by extroonie
```suggestion
    this.guildID = guildID ?? null;
```